framework/src/modules/chain/dpos/delegates.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+const { hash } = require('@liskhq/lisk-cryptography');
+
+const shuffleActiveDelegateList = (round, list) => {
```

Comment:
Maybe `shuffleDelegateListForRound` makes more sense since it's not necessary that this function is only called for current active round.